plugins/generator-1.20.1/forge-1.20.1/templates/livingentity/livingentity.java.ftl

```diff
@@ -41,7 +41,7 @@ import net.minecraft.network.syncher.EntityDataAccessor;
 
 <#assign extendsClass = "PathfinderMob">
 
-<#if data.aiBase != "(none)">
+<#if data.aiBase != "(none)" && data.mobBehaviourType != "Raider">

```

Comment:
Why wouldn't you set the AI entry to none when the rider behavior type is selected?

You deselect breedable and handle it in UI, why is those part handled differently? It is inconsistent.

Please handle this in UI and rever this change

Reply:
Done